matt-solution.js

```diff
@@ -0,0 +1,11 @@
+var sum = 0;
+function oddNumbers() {
+	for (var i = 0; i < 1000; i++) {
```

Comment:
sum is not set at start so if this were run twice it would be incorrect until eventual overflow